ddp-workspace/projects/ddp-sdk/src/lib/components/user/manageParticipants.component.ts

```diff
@@ -77,7 +79,10 @@ export class ManageParticipantsComponent implements OnDestroy {
     }
 
     public add(): void {
-        this.serviceAgent.add(this.currentParticipant).pipe(
+        this.serviceAgent.addParticipant(
+          this.config.studyGuid,
+          { alias: this.currentParticipant },
```

Comment:
From PR https://github.com/broadinstitute/ddp-study-server/pull/463, it looks like this API doesn't accept `alias` in the request payload. Even if it did, maybe setting it to the current participant's guid is not best way? The alias is meant to be kind of a "name" for the new governed participant so setting it to current participant is a bit odd.

Reply:
You are correct. This part of code was not written by me so I decided to change it so it behaves the same way it did before our changes. Anyway, I'll remove the part with alias.

Reply:
We decided to keep this part, @YuriyPavlovBI will handle alias field if it is provided.

Reply:
To clarify, we are not using this exact component in our AT study. These changes were made only to ensure that this component behaves same way as it did before.

Reply:
I don't think it should pass in the current participant guid. That doesn't make sense. Let's keep it same for now and I'll add ticket to fix this separately.